core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceTypeUtilTest.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.internal.server.annotation;
+
+import static com.linecorp.armeria.internal.server.annotation.AnnotatedServiceTypeUtil.stringToType;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.UUID;
+
+import org.junit.jupiter.api.Test;
+
+public class AnnotatedServiceTypeUtilTest {
```

Comment:
I think you can remove all `public` keywords from that class.

Reply:
Unfortunately, no, as then every inner class will be "kind of" implicit package private (ie, unless you use reflections), which will make the constructors also package private, which will make checkstyle complain. So either:
1. Test class is public
2. The classes only used for these tests are moved into separate files.

I figured adding the public keyword was the easier and cleaner

Reply:
Oops... sorry for the confusion. Please ignore my comment.

Reply:
No worries. I fought checkstyle for a long time until I understood why it was complaining. (initially I wrote it without the `public` keyword)

Reply:
You might want to use `@SuppressWarnings("CheckStyleRuleName")` to suppress that error, but I guess either ways are fine.